util/kubeconfig/kubeconfig.go

```diff
@@ -42,8 +46,8 @@ func FetchKubeconfigFromSecret(c client.Client, cluster *clusterv1.Cluster) ([]b
 	return data, nil
 }
 
-// NewKubeconfig creates a new Kubeconfig where endpoint is the ELB endpoint.
-func NewKubeconfig(clusterName, endpoint string, caCert *x509.Certificate, caKey *rsa.PrivateKey) (*api.Config, error) {
+// New creates a new Kubeconfig where endpoint is the ELB endpoint.
```

Comment:
I think ELB isn't quite right, it's a single APIServer endpoint that may be an ELB or a direct connection to a control plane